DOCSUP-14092: Documented the settings for inserting nested (#28666)
ClickHouse/ClickHouse: docs/en/interfaces/formats.md

$ cat {filename} | clickhouse-client --query="INSERT INTO {some_table} FORMAT Parquet"
```

> To insert `Array(Struct)` values into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns you must switch on the [input_format_parquet_import_nested](../operations/settings/settings.md#input_format_parquet_import_nested) setting.

Reviewer: ```suggestion
To insert data into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) column as an array of structs you must switch on the [input_format_parquet_import_nested](../operations/settings/settings.md#input_format_parquet_import_nested) setting.
```

$ cat filename.arrow | clickhouse-client --query="INSERT INTO some_table FORMAT Arrow"
```

> To insert `Array(Struct)` columns into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns use the setting [input_format_arrow_import_nested](../operations/settings/settings.md#input_format_arrow_import_nested).

Reviewer 2: ```suggestion
To insert `Array(Struct)` values into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns you must switch on the [input_format_parquet_import_nested](../operations/settings/settings.md#input_format_parquet_import_nested) setting.
```

> To insert `Array(Struct)` columns into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns use the setting [input_format_arrow_import_nested](../operations/settings/settings.md#input_format_arrow_import_nested).

Reviewer 2: ```suggestion
To insert `Array(Struct)` values into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns you must switch on the [input_format_arrow_import_nested](../operations/settings/settings.md#input_format_arrow_import_nested) setting.
```

$ cat filename.orc | clickhouse-client --query="INSERT INTO some_table FORMAT ORC"
```

> To insert `Array(Struct)` columns into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns use the setting [input_format_orc_import_nested](../operations/settings/settings.md#input_format_orc_import_nested).

Reviewer 2: ```suggestion
To insert `Array(Struct)` values into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns you must switch on the [input_format_orc_import_nested](../operations/settings/settings.md#input_format_orc_import_nested) setting.
```
